.travis.yml

```diff
@@ -9,3 +9,4 @@ python:
 # install: "sudo apt-get install"
 # command to run tests, e.g. python setup.py test
 script: python cpplint_unittest.py
+script: python nitpick_unittest.py
```

Comment:
@eyalyowza - this is the breaking line

side note - it was somewhat annoying to get to the diff view for line-level comment